application/build.gradle

```diff
@@ -1,12 +1,32 @@
 plugins {
-    id 'org.springframework.boot' version '2.7.8'
+	id 'org.springframework.boot' version '2.7.8'
+	id 'com.google.cloud.tools.jib' version '3.3.1'
 }
 
 dependencies {
-    implementation 'org.springframework.boot:spring-boot-starter-web'
-    implementation project(':infrastructure-petstore-rest-client')
-    implementation project(':domain')
+	implementation 'org.springframework.boot:spring-boot-starter-web'
+	implementation 'org.springframework.boot:spring-boot-starter-actuator'
+	implementation project(':infrastructure-petstore-rest-client')
+	implementation project(':infrastructure-pet-jms')
+	implementation project(':domain')
 
-    developmentOnly 'org.springframework.boot:spring-boot-devtools'
-    testImplementation 'org.springframework.boot:spring-boot-starter-test'
-}
\ No newline at end of file
+	testImplementation 'org.springframework.boot:spring-boot-starter-test'
+	testImplementation 'org.awaitility:awaitility:4.2.0'
```

Comment:
Can we centralise these versions? I guess, we need the libraries in other modules as well

Reply:
We could provide it to all subprojects. But we can also wait until it really needed.